OpenAL32/alSource.cpp

```diff
@@ -2703,9 +2706,12 @@ AL_API ALvoid AL_APIENTRY alSourcePlayv(ALsizei n, const ALuint *sources)
             [&context](ALuint sid) -> void
             {
                 ALsource *source{LookupSource(context.get(), sid)};
-                source->OffsetType = AL_NONE;
-                source->Offset = 0.0;
-                source->state = AL_STOPPED;
+                if(source)
```

Comment:
These also can't be null since the IDs were looked up earlier in the call and verified to be non-null (the source lock is also being held so a call on another thread can't delete the ID in between). If it manages to be null, an error needs to be set before any modification happens to the sources.